### PR TITLE
fix: PDF display request in v3

### LINKED ISSUE #3826: [Bug] PDFs Won't Load From DCM4CHEE
### Describe the Bug

If I don't specify "pdf" in "singlepart" field of config then when trying to load a pdf I get a 406 Not Accepted Error. The request url is bulkdata and the accept header being sent is multi-part/related type: application/octet-stream. If I recreate the request in postman I get this response {"errorMessage":"Not accepted instances present."}, however if I used  multi-part/related type: application/pdf in postman, I get a 200 response. It doesn't seem like dicomweb-client bulkdata requests ever use that header though? 

If instead I specify "pdf" in "singlepart" instead a rendered endpoint is passed to the Object element and the pdf still fails to load. If I hit this rendered endpoint in postman I also get a 200 response. I don't see an actual request being made to the rendered endpoint in dev tools though.

We are using DCM4CHEE 31.1.

### Steps to Reproduce

1. Choose one of the config options above.
2. Double click DOC modality pdf series.

### The current behavior

"No online PDF viewer installed"

### The expected behavior

PDF loads.

### OS

Windows 10

### Node version

16.17.0

### Browser

119.0.6045.160

### PR DESCRIPTION
**Pull request by wayfarer3130**
fixes #3826 

### Context

Some PACS systems required requesting encapsulated PDF using multipart/related;type=application/pdf instead of application/octet-stream.  This PR enables the required fetch to be completed.

### Changes & Results

Change the retrieveBulkdata to use the dicomweb-client 0.10.4 version which allows fetching bulkdata using application/pdf, with a backup of application/octet-stream

### Testing

Find an encapsulated PDF instance on a DCM4CHEE back end and ensure it displays in OHIF

#### PR

- [] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [] My code has been well-documented (function documentation, inline comments,
  etc.)

#### Public Documentation Updates

- [] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [] OS: 
- [] Node version: 
- [] Browser:
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

File: extensions/default/src/DicomWebDataSource/index.js [hunk 1]
```diff
@@ -388,9 +389,13 @@ function createDicomWebApi(dicomWebConfig, servicesManager) {
                 // isn't well specified in the standard, but is needed in
                 // any implementation that stores static copies of the metadata
                 StudyInstanceUID: naturalized.StudyInstanceUID,
+                mediaTypes: mediaType
```
Comment (wayfarer3130) [inline extensions/default/src/DicomWebDataSource/index.js:392]: This change allow requesting first in the provided media type, and then defaulting to application/octet-stream if not found.

File: platform/app/public/config/e2e.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ window.config = {
         supportsFuzzyMatching: false,
         supportsWildcard: true,
         staticWado: true,
-        singlepart: 'bulkdata,video,pdf',
+        singlepart: 'video',
```
Comment (sedghi) [inline platform/app/public/config/e2e.js:56]: should we change other servers as well? like dcm4chee one?

**Comment by sedghi** [comment]
@Michael-Andersen Can you check if this branch solve the issue for your PDF reported in #3826 . Thanks

**Comment by Michael-Andersen** [comment]
@sedghi Yes pdfs are loading properly from my DCM4CHEE instance on this branch, thanks!

**Review by sedghi — APPROVED** [review]

**Comment by StefanSmaczny** [comment]
Hi,

does this fix also apply to Orthanc? I mentioned the same problem with displaying PDFs in the Orthanc Forum https://discourse.orthanc-server.org/t/ohif-plugin-is-displaying-mri-images-but-not-pdf-or-sr/4315/10 , but was told to come here.
Also, it seems that I am not the only one with this problem:
https://community.ohif.org/t/pdf-load-solution-for-ohif-orthanc/1224

Any advice would be highly appreciated! Please forgive me if this trivial and can be solved easily, but I do not have a lot of experience with OHIF (or Orthanc).

File: platform/app/public/config/e2e.js (hunk 2 above)
Comment (holakunle) [reply platform/app/public/config/e2e.js:56]: hello @sedghi please do am experiencing the same issue (no online PDF viewer installed) with orthanc .

**Comment by holakunle** [comment]
hello @StefanSmaczny  have you been able to find a way, am experiencing the same issues with orthanc

**Comment by sedghi** [comment]
@StefanSmaczny what is the current issue? do you have a issue number?

**Comment by holakunle** [comment]
> @StefanSmaczny what is the current issue? do you have a issue number?

Hello @sedghi 

Ohif + orthanc isn't displaying PDFs. It says " No pdf viewer installed". Have been trying to find a way round it from the above solution but I guess it's specifically applies to DCM4CHEE. Any help on this will be much appreciated. 

Thank you

**Comment by amazy** [comment]
@sedghi here is an issue number https://github.com/OHIF/Viewers/issues/4256, created by @holakunle and updated by me.

**Pull request merged**